cog/daily_charge.py

```
@@ -19,7 +19,7 @@
 
 def get_channels():  # 取得特殊用途頻道的清單，這裡會用來判斷是否在簽到頻道簽到，否則不予受理
     with open(
-        f"{os.getcwd()}/DataBase/server.config.json", "r", encoding="utf-8"
+        f"{os.getcwd()}/database/server.config.json", "r", encoding="utf-8"
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)